packages/mask/src/components/shared/SelectRecipients/useTwitterIdByWalletSearch.tsx

```diff
@@ -0,0 +1,49 @@
+import { EMPTY_LIST, NextIDPersonaBindings, NextIDPlatform, ProfileIdentifier } from '@masknet/shared-base'
+import { isValidAddress } from '@masknet/web3-shared-evm'
+import { uniq } from 'lodash-unified'
+
+export function useTwitterIdByWalletSearch(bindings: NextIDPersonaBindings[] | undefined, value: string) {
+    if (!bindings) return EMPTY_LIST
+    const type = isValidAddress(value)
```

Comment:
plz dot not abuse nested ternary operator

Reply:
ack